relay-server/src/endpoints/project_configs.rs

```diff
@@ -130,7 +130,13 @@ async fn inner(
     let mut configs = HashMap::with_capacity(keys_len);
     let mut pending = Vec::with_capacity(keys_len);
     let global_config = match inner.global {
-        true => Some(state.global_config().send(global_config::Get).await?),
+        true => Some(
+            state
+                .global_config()
+                .send(global_config::Get)
+                .await?
+                .get_ready_or_default(),
```

Comment:
So here we always actually return the default if the config currently unavailable?

Reply:
yes, not because i think it's good but because It's what we currently do and I wasn't sure to add another behaviour in the scope of this PR

Reply:
Does this mean that PoP relays can get a global config and process envelopes when they should not? A processing relay that hasn't fetched a global config from Sentry will send the default config to PoP relays, and PoPs will start processing. This is also the case for the following relays in the chain.

Reply:
For others: me and iker synced and concluded that it's not a problem in practice because we deploy processing before pops with a significant time delay, and by that time processing should have a global config ready.

please others come with feedback about this

Reply:
The time delay does not matter, if the processing Relays cannot get the global config from upstream. 
When the PoPs are deployed while there is an incident with fetching global config, PoPs will get default one.

Reply:
youre right, its not ideal. I discussed this with @Dav1dde  who also pointed this out. We would have to communicate to the downstream relays that it's a global config they're receiving, or not send back at all. Backwards compatibility is an issue here.

Reply:
I think this still mirrors current behaviour, we can tackle this in a separate issue imo. The possible solutions are actually quite tricky and depends on wether we want to change the API or not.